shared/utils/localApiUrl.js

```diff
@@ -3,10 +3,9 @@ import config from './config';
 /*
  * Resolves the api url for the client or server
  */
-export default function getLocalApiUrl() {
-  return typeof window === 'undefined'
-    // Server
-    ? config('localApiUrl')
-    // Client
-    : '/api';
-}
+export default
+typeof window === 'undefined'
```

Comment:
Don't you have any lint issue with the indentation here?

Reply:
No, but I got an error when I had two spaces